src/Resources/contao/templates/backend/be_login.html5

```diff
@@ -81,7 +81,7 @@
         <p>Contao Open Source CMS :: Copyright ©2005-<?= date('Y') ?> by Leo Feyer :: Extensions are copyright of their respective owners :: Visit <a href="https://contao.org" target="_blank">contao.org</a> for more information :: Obstructing the appearance of this notice is prohibited by law!</p>
         <p>Contao is distributed in the hope that it will be useful but WITHOUT ANY WARRANTY. Without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details. Contao is free software. You can redistribute it and/or modify it under the terms of the GNU/LGPL as published by the Free Software Foundation.</p>
       </div>
-      <p id="go_to_frontend"><a href="<?= $this->route('contao_root') ?>" class="footer_preview" title="<?= $this->feLink ?>"><?= $this->feLink ?></a></p>
```

Comment:
It does not really have many ramifications. The existing ones are fixed.
It's a bit strange, but it's the best solution I could come up with. And I think it's actually correct, because `/` is not a valid URL if you have `prefix_locale` enabled. In cases like the backend login link, we're not actually linking to a known route, we're linking to the installation base path.